docs/book/GLOSSARY.md

```diff
@@ -128,6 +142,12 @@ A generically understood combination of a kernel and system-level userspace inte
 
 # P
 
+## Pivot
+
+Pivot is a process for moving the provider components and declared cluster-api resources from a Source cluster to a Target cluster.
```

Comment:
Let's update this to `Source Management cluster` and `Target Management cluster`